packages/config/config/devices/0x0371/zwa012.json

```diff
@@ -47,52 +63,142 @@
+			"defaultValue": 1
 		},
 		"5": {
-			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_trigger"
+			"$import": "~/0x0086/templates/aeotec_template.json#range_test_double_click"
 		},
 		"6": {
-			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_command_type"
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_trigger"
 		},
 		"7": {
-			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_basic_set_on"
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_command_type"
 		},
 		"8": {
-			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_basic_set_off"
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_basic_set_on"
 		},
 		"9": {
-			"$import": "~/0x0086/templates/aeotec_template.json#time_delay_on"
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_basic_set_off",
+			"defaultValue": 0
 		},
 		"10": {
-			"$import": "~/0x0086/templates/aeotec_template.json#time_delay_off"
+			"$import": "~/0x0086/templates/aeotec_template.json#time_delay_on",
+			"defaultValue": 0
 		},
 		"11": {
-			"$import": "~/0x0086/templates/aeotec_template.json#time_delay_tamper"
+			"$import": "~/0x0086/templates/aeotec_template.json#time_delay_off",
+			"defaultValue": 0
 		},
 		"12": {
-			"$import": "~/0x0086/templates/aeotec_template.json#report_on_tamper_cancel"
+			"$import": "~/0x0086/templates/aeotec_template.json#time_delay_tamper",
+			"defaultValue": 0
 		},
 		"13": {
-			"$import": "~/0x0086/templates/aeotec_template.json#central_scene_functionality"
+			"$import": "~/0x0086/templates/aeotec_template.json#report_on_tamper_cancel"
 		},
 		"14": {
-			"$import": "~/0x0086/templates/aeotec_template.json#tilt_sensor"
+			"$import": "~/0x0086/templates/aeotec_template.json#central_scene_functionality"
 		},
 		"15": {
+			"$import": "~/0x0086/templates/aeotec_template.json#tilt_sensor"
+		},
+		"16": {
+			"$import": "~/0x0086/templates/aeotec_template.json#invert_state_report",
+			"label": "Tilt sensor polarity when sensor is vertical",
+			"options": [
+				{
+					"label": "OFF",
+					"value": 0
+				},
+				{
+					"label": "ON",
+					"value": 1
+				}
+			]
+		},
+		"17": {
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_trigger",
+			"label": "Association Group 5: Trigger"
+		},
+		"18": {
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_command_type",
+			"label": "Association Group 5: Command Type"
+		},
+		"19": {
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_basic_set_on",
+			"label": "Association Group 5: Basic Set Value (On)"
+		},
+		"20": {
+			"$import": "~/0x0086/templates/aeotec_template.json#association_group_2_basic_set_off",
+			"label": "Association Group 5: Basic Set Value (Off)",
+			"defaultValue": 0
+		},
```

Comment:
@blhoward2 what do you think? I'm not sure if importing from a "group 2" template and then changing everything to a 5 is better than just importing the original `"~/templates/master_template.json#base_0-255_nounit"`

Reply:
I'm fine either way. Where the ones for other groups get repeated over devices I've also templated those in the past just so it's easier to rename them all should we ever want to do so.

Reply:
okay, we can keep these then.